contrib/grimshot

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is bash really necessary? Given the variety of users of this, I'm sure there's people who won't have it installed (e.g.: some alpine and BSD users).

(in any case, this should be `/usr/bin/env bash`)

Reply:
Yeah, you are right, but I can't think of a shebang for something like "POSIX sh-based shell supporting arrays" (see [SC3054](https://github.com/koalaman/shellcheck/wiki/SC3054))